src/js/_enqueues/wp/view-transitions.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * @output wp-includes/js/wp-view-transitions.js
+ */
+
+window.wp = window.wp || {};
+window.wp.viewTransitions = {};
+
+/**
+ * Initializes view transitions for the current URL.
+ *
+ * @param {object}  config                         The view transitions configuration.
+ * @param {string}  config.postSelector            General selector for post elements in the DOM.
+ * @param {object}  config.globalTransitionNames   Map of selectors for global elements (queried relative to 'body')
+ *                                                 and their view transition names.
+ * @param {object}  config.postTransitionNames     Map of selectors for post elements (queried relative to an element
+ *                                                 identified by config.postSelector) and their view transition names.
+ * @param {boolean} config.chronologicalSlideInOut Whether slide in/out animation for chronological URL relationship
+ *                                                 (date- or pagination-based) should be enabled.
+ */
+window.wp.viewTransitions.init = ( config ) => {
+	if ( ! window.navigation || 'CSSViewTransitionRule' in window === false ) {
```

Comment:
Suggestion by PhpStorm to simplify this:

```suggestion
	if ( ! window.navigation || ! ( 'CSSViewTransitionRule' in window ) ) {
```